src/test/java/io/confluent/idesidecar/restapi/auth/CCloudOAuthContextTest.java

```diff
@@ -105,6 +106,83 @@ void createTokensFromAuthorizationCodeShouldExchangeControlAndDataPlaneTokens()
     }
   }
 
+  @Test
+  void exchangeControlPlaneTokenShouldFailIfCookieHeaderIsEmpty() throws Throwable {
+    var testContext = new VertxTestContext();
+    var authContext = new CCloudOAuthContext();
+
+    wireMock.register(
+        WireMock
+            .post("/api/sessions")
+            .willReturn(
+                WireMock
+                    .aResponse()
+                    .withHeader("Set-Cookie", "")
```

Comment:
this looks a little different from the original test. while re-writing, I realized we weren't checking for a completely empty header, so I updated the code here to catch it.